internal/pkg/api/server/common/extension/dependencies/dependencies.go

```diff
@@ -57,23 +57,23 @@ func RegisterPlugin(pkgPath string) {
 					case "endpoint-method":
 
 						search := `
-	{{- if .ServerStream }}
+{{- if .ServerStream }}
```

Comment:
@jprochazk mas to cele dobre :+1: , len si odsadil sablonu, ... v `replace` by to az tak nevadilo, kedze vygenerovany Go kod sa este formatuje, ... ale v `search` to vadi.

Mozes to mergnut do svojho PR, ak chces, a tento draft zavriet.